github/release_notes/model.py

```diff
@@ -71,7 +71,9 @@ class ReleaseNote:
     user_login: str
     is_current_repo: bool
     from_same_github_instance: str
-    cn_source_repo: str
+    current_component_name: str
```

Comment:
how about adding the full component (which would include the access), rather than merely the name?